sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -325,36 +324,54 @@ def test_lassocv_alphas_validation(alphas, err_type, err_msg):
         lassocv.fit(X, y)
 
 
-def _scale_alpha_inplace(estimator, n_samples):
-    """Rescale the parameter alpha from when the estimator is evoked with
-    normalize set to True as if it were evoked in a Pipeline with normalize set
-    to False and with a StandardScaler.
-    """
-    if ("alpha" not in estimator.get_params()) and (
-        "alphas" not in estimator.get_params()
-    ):
-        return
-
-    if isinstance(estimator, (RidgeCV, RidgeClassifierCV)):
-        # alphas is not validated at this point and can be a list.
-        # We convert it to a np.ndarray to make sure broadcasting
-        # is used.
-        alphas = np.asarray(estimator.alphas) * n_samples
-        return estimator.set_params(alphas=alphas)
-    if isinstance(estimator, (Lasso, LassoLars, MultiTaskLasso)):
-        alpha = estimator.alpha * np.sqrt(n_samples)
-    if isinstance(estimator, (Ridge, RidgeClassifier)):
-        alpha = estimator.alpha * n_samples
-    if isinstance(estimator, (ElasticNet, MultiTaskElasticNet)):
-        if estimator.l1_ratio == 1:
-            alpha = estimator.alpha * np.sqrt(n_samples)
-        elif estimator.l1_ratio == 0:
-            alpha = estimator.alpha * n_samples
-        else:
-            # To avoid silent errors in case of refactoring
-            raise NotImplementedError
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.ConvergenceWarning")
```

Comment:
There was a conflict which I resolved, but now this new test has appeared in the diff. I'm confused by that because it looked like this was a change from `main` but if that is true, why does it show up in the diff?

@lorentzenchr does it make sense to you or can you resolve the conflict (locally)? Sorry for the mess :-/

Reply:
I think we should just delete this test again. I can't find it in https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/linear_model/tests/test_coordinate_descent.py so we don't need it. Still puzzled why it showed up when resolving the conflict